### PR TITLE
Bump version @nextcloud/vue to 8.12.0

### PR DESCRIPTION
**Pull request by SagarGi**
## Description

This PR:
- Bumps @nextcloud/vue  library from `7.0.0` to `8.12.0`
- Bumps other related libraries in the application.

## Related Issue or Workpackage

- Related Issue (https://community.openproject.org/projects/nextcloud-integration/work_packages/55650)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [x] Updated `CHANGELOG.md` file

### CODE REVIEW

**Comment by SagarGi** [comment]
But this needs a manual testing in different version of nextcloud (26,27,28,29) if there is inconsistencies in the UI especially.
CC @nabim777 @individual-it

File: src/components/AdminSettings.vue [hunk 1]
```diff
@@ -766,7 +763,7 @@ export default {
 						await this.clearOPOAuthClientValues()
 					}
 				},
-				true
+				true,
```
Comment (individual-it) [inline src/components/AdminSettings.vue:766]: do we want to have all those small format changes, or have an extra PR?

Comment (SagarGi) [reply src/components/AdminSettings.vue:766]: We have to in this PR, its the updated style lint thing

**Review by individual-it — APPROVED** [review]

**Comment by SagarGi** [comment]
The smoke testing seems to be fine with one bug https://community.openproject.org/projects/nextcloud-integration/work_packages/56807 which only occurs in local development setup. But in production it seems to be working. And all the smoke test is passing in `OpenProject` version 14.3.1 But we can still merge this since there will be another QA from the OpenProject QA team.

**Pull request merged**